Gemfile

```diff
@@ -43,6 +46,9 @@ group :test do
   gem 'simplecov', require: false
   gem 'coveralls', require: false
   gem 'database_cleaner'
+  gem 'vcr'
+  gem 'webmock'
+  gem 'faraday'
```

Comment:
We've already got httparty - maybe use it instead of adding a new gem.

Reply:
Good point.